navigation_onboarding/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include "main.hpp"
+
+bool doNavigation(){
+  navigation::Navigation nav;
+  bool stop_yet = nav.navigate();
+  return stop_yet;
```

Comment:
You don't mention this function in the README so I assume the reader shouldn't touch these. 

```cpp
return nav.navigate();
```